Mathlib/LinearAlgebra/BilinearForm/Hom.lean

```diff
@@ -60,14 +60,18 @@ def toLinHomAux₁ (A : BilinForm R M) (x : M) : M →ₗ[R] R := A x
 #align bilin_form.to_lin_hom_aux₁ LinearMap.BilinForm.toLinHomAux₁
 
 /-- Auxiliary definition to define `toLinHom`; see below. -/
+@[deprecated]
```

Comment:
Here is how adding a date could look like:
```suggestion
@[deprecated] -- 2024-04-26

Reply:
@grunweg there was some discussion around this [here](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Next.20steps.20for.20BilinForm) but it didn't appear to reach a definite conclusion.

Is your suggestion that the date should be the date of deprecation, or the date when the result can be deleted from Mathlib?

Reply:
I would aim for the date of deprecation: a deprecation policy will imply then this can be deleted.

Fair enough; I have started [a zulip thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Deprecation.20policy) trying to create consensus.